test/common/router/router_test.cc

```diff
@@ -1336,47 +1336,6 @@ TEST_F(RouterTest, RetryUpstreamPerTryTimeout) {
   EXPECT_TRUE(verifyHostUpstreamStats(1, 1));
 }
 
-TEST_F(RouterTest, RetryUpstreamPerTryTimeoutNoUpstreamHost) {
```

Comment:
Just to double-check, this is the test just added in #4904 merged one day ago, was this supposed to be removed in #4905?

Reply:
Ah you're right. Turns out I added a test relying on behavior that I changed a day later

Reply:
Let me fix the test instead

Reply:
@snowp do you have an ETA? Would like to unbreak master ASAP.

Reply:
already done in the latest commit

Reply:
should be ready to merge